code/API_definitions/qod-api.yaml

```diff
@@ -63,41 +63,41 @@ paths:
                     code: INVALID_ARGUMENT
                     message: "Schema validation failed at  ..."
                 MsisdnRequired:
```

Comment:
Should we update this line too?

Reply:
I would say so.

Reply:
Done in https://github.com/camaraproject/QualityOnDemand/pull/129/commits/ed3bbdc954ff21982ceee48b840c6d44d8aa9dec